src/Document.php

```diff
@@ -1805,38 +1805,21 @@ public static function isImage($file)
     }
 
     /**
-     * Get image path for a specified context.
+     * Get image path.
      * Will call image resize if needed.
      *
      * @since 9.2.1
      *
      * @param string  $path    Original path
-     * @param string  $context Context
      * @param integer $mwidth  Maximal width
      * @param integer $mheight Maximal height
      *
      * @return string Image path on disk
      */
-    public static function getImage($path, $context, $mwidth = null, $mheight = null)
+    public static function getImage($path, $mwidth = null, $mheight = null)
```

Comment:
I would be safer to keep the previous `getImage()` method signature, deprecate it, and declare a new `public static function getResizedImagePath(string $path, int $width, int $height): string` method.

Reply:
done !